test/unit/Util/Autoload/ClassLoaderTest.php

```diff
@@ -72,19 +75,18 @@ public function testAddClassThrowsExceptionWhenClassAlreadyRegisteredInAutoload(
 
         $this->expectException(ClassAlreadyRegistered::class);
         $loader->addClass($reflection);
-
-        spl_autoload_unregister($loader);
     }
 
+    /**
+     * @runInSeparateProcess
+     */
     public function testAddClassThrowsExceptionWhenClassAlreadyLoaded(): void
     {
         $loaderMethod = $this->createMock(LoaderMethodInterface::class);
         $loader       = new ClassLoader($loaderMethod);
 
         $this->expectException(ClassAlreadyLoaded::class);
         $loader->addClass(ReflectionClass::createFromName(stdClass::class));
-
-        spl_autoload_unregister($loader);
```

Comment:
Probably to be kept in the same process?

Reply:
The call is not called because of thrown exception.